src/theme/theme.ts

```diff
@@ -0,0 +1,23 @@
+import type {ColorModeOptions} from '../core/color-mode/type'
+import base from './base'
+
+const config: ColorModeOptions = {
+  useSystemColorMode: false,
+  initialColorMode: 'light',
```

Comment:
I think the better way would be to define an enum for this

Reply:
but this initialColorMode is use ColorMode's type